packages/file/index.js

```diff
@@ -1,13 +1,34 @@
-let { createGzip } = require('zlib')
+let chalk = require('chalk')
+let { gte } = require('semver')
+let { constants, createBrotliCompress, createGzip } = require('zlib')
 let { promisify } = require('util')
 let fs = require('fs')
 
 let stat = promisify(fs.stat)
+let brotliRequiredNodeVersion = 'v11.7.0'
```

Comment:
It should be `const BROTLI_NODE_VERSION`

Reply:
fixed